maplibre-native-ios/Amazon Location Service Demo/AWSSignatureV4Delegate.swift

```diff
@@ -4,67 +4,138 @@
 import AWSCore
 import Mapbox
 
-class AWSSignatureV4Delegate : NSObject, MGLOfflineStorageDelegate {
-    private let region: AWSRegionType
-    private let identityPoolId: String
-    private let credentialsProvider: AWSCredentialsProvider
-
-    init(region: AWSRegionType, identityPoolId: String) {
-        self.region = region
-        self.identityPoolId = identityPoolId
-        self.credentialsProvider = AWSCognitoCredentialsProvider(regionType: region, identityPoolId: identityPoolId)
-        super.init()
+struct MapboxConfig {
```

Comment:
Can we rename this to `MapLibreConfig` (and reference MapLibre instead everywhere else, since that more accurately describes this sample)?